### PR TITLE
Improve Rerun Menu with link to Rerun Discord

### PR DESCRIPTION
**Pull request by emilk**
### What
Cleans up the code around the options menu, and splits out the debug options to its own menu.

Also adds a link to our Discord (part of https://github.com/rerun-io/rerun/issues/4436)

The icon comes from https://discord.com/branding

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/4661/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/4661/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/4661/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4661)
- [Docs preview](https://rerun.io/preview/c3f224b67f947b1434d236757fe77c01c5efb562/docs) 
- [Examples preview](https://rerun.io/preview/c3f224b67f947b1434d236757fe77c01c5efb562/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: crates/re_ui/data/icons/discord.png
Comment (abey79) [inline crates/re_ui/data/icons/discord.png:1]: Might be nice to make it 14x14 for consistence with all other icons that I'm currently updating.

Comment (abey79) [reply crates/re_ui/data/icons/discord.png:1]: Maybe @martenbjork want to take a stab at it, too :)

**Review by abey79 — APPROVED** [review]
Nice 👍🏻

File: crates/re_ui/data/icons/discord.png
Comment (emilk) [reply crates/re_ui/data/icons/discord.png:1]: 14x14 what, pixels or ui units?

if it is 14x14 ui units, then we should probably make it 28x28 pixels, right?

Comment (emilk) [reply crates/re_ui/data/icons/discord.png:1]: Anyways, I chose the 24 width because it is 2x the currently default icon size of 12x12

Comment (abey79) [reply crates/re_ui/data/icons/discord.png:1]: Right, actually I meant 28x28, which is the new default.

Comment (emilk) [reply crates/re_ui/data/icons/discord.png:1]: Let's fix it later :)

**Pull request merged**